docs/readme.md

```diff
@@ -7,9 +7,11 @@
 You may wish to launch a local version of our documentation website
 to test changes for formatting, link resolution, etc.
 
-Python 3 is required:
+Python 3 is required. It is also good practice to use virtualenv:
```

Comment:
This is half-done by `make` 
```sh
make docs
. derived/.venv/docs/bin/activate && (  cd derived/docs; python3 -m mkdocs serve  )
```

Reply:
Ah! I didn't know that make did this. I can update that.